icu/plan.sh

```diff
@@ -11,8 +11,9 @@ EOF
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=("Unicode-TOU")
 # shellcheck disable=SC2059
-pkg_source="http://download.icu-project.org/files/icu4c/${pkg_version}/icu4c-$(printf "%s" "$pkg_version" | tr . _)-src.tgz"
+pkg_source="https://github.com/unicode-org/icu/releases/download/release-$(printf "%s" "$pkg_version" | tr . -)/icu4c-$(printf "%s" "$pkg_version" | tr . _)-src.tgz"
```

Comment:
Since plans already depend on bash-specific features, if you wanted you could do the replacement in bash:

```
${pkg_version//./-}
```